src/main/kotlin/no/nav/klage/oppgave/service/ElasticsearchService.kt

```diff
@@ -186,15 +185,41 @@ open class ElasticsearchService(
 
         addSecurityFilters(baseQuery)
 
+        val combinedInnerFnrAndTemaQuery = QueryBuilders.boolQuery()
+        baseQuery.must(combinedInnerFnrAndTemaQuery)
+
+        val innerFnrAndTemaQuery = QueryBuilders.boolQuery()
+        combinedInnerFnrAndTemaQuery.should(innerFnrAndTemaQuery)
+
```

Comment:
Vet ikke om det bare er meg, men jeg synes det er litt vanskelig å lese rekkefølgen her. D.v.s. at de først blir "brukt", og så "implementert". Kan godt være at dette blir mest naturlig. Bare noe jeg lurer på :)

Reply:
Jeg lurer på om jeg drøyer litt med å gjøre noe med denne. Jeg tror kanskje vi skal knekke om litt på hele søke-implementasjonen, ref diskusjonen jeg la opp til på Slack forleden. At vi f.eks lager mindre metoder for hvert felt man skal søke på, og så kaller de metodene i ulike usecase. Da tror jeg det vil bli enklere å lese dette, og da tenker jeg at det kan være greit å gjøre det du etterspør her også.